test/node-unit/cli/options.spec.js

```diff
@@ -294,7 +295,6 @@ describe('options', function () {
 
             result = loadOptions('--no-diff --no-config');
           });
-
           it('should return parsed args, default config and package.json', function () {
```

Comment:
[Style] Nit: unrelated change?

```suggestion

          it('should return parsed args, default config and package.json', function () {
```